dali/util/make_string_test.cc

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+#include "dali/core/geom/vec.h"
+#include "dali/util/make_string.h"
+
+namespace dali {
+
+TEST(MakeStringTest, default_delimiter) {
+  auto str = make_string("jeden", 2, 3);
+  ASSERT_EQ(str, "jeden 2 3");
+}
+
+TEST(MakeStringTest, custom_delimiter) {
+  auto str = make_string_delim("]:->", "jeden", 2, 3);
+  ASSERT_EQ(str, "jeden]:->2]:->3");
+}
+
```

Comment:
Please add some tests for boundary conditions:
* no arguments
* one argument
* only delimiter
* delimiter + one argument.